### PR TITLE
 feat: add guess_config

### PR DESCRIPTION
**Pull request by cjquines**
add parameter `guess_config` to guess endpoints on stainless config after updating openapi spec

tested against staging, prod

### CODE REVIEW

**Comment by meorphis** [comment]
were you able to test this in prod? you can create a test repo to host your spec and add the github action like so:
```
jobs:
  stainless:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4
      - uses: stainless-api/upload-openapi-spec-action@{your branch's SHA}
```

**Comment by cjquines** [comment]
how fast does prod update? i ran an action (https://github.com/stainless-sdks/cjquines-source/actions/runs/10374835174/job/28723176533) but it doesn't look like it did the thing

**Comment by cjquines** [comment]
ok cool, here's [action run](https://github.com/stainless-sdks/cjquines-source/actions/runs/10377728098/job/28732540674) and [corresponding commit](https://github.com/stainless-sdks/cjquines-config/commit/6bcddfd4cf9fc463d7893019bd62f95f6467af97)

**Review by meorphis — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -53,6 +54,8 @@ You can identify your Stainless project name on the [Stainless dashboard](https:
 
 `commit_message` is an optional parameter that allows you to specify the commit message that we will use for the commits generated for your SDKs as a result of your API change (and which will subsequently appear in the Changelog). If you provide it, please use the [Conventional Commits format](https://www.conventionalcommits.org/en/v1.0.0/) for all of your messages. If you do not provide it, we will use a default.
 
+`guess_config` is an optional parameter that, when `true`, will regenerate the endpoints in the Stainless config file. By default, it is `false`.
```
Comment (meorphis) [inline README.md:57]: actually maybe something like:

> `guess_config` is an optional parameter that, when `true`, will update your Stainless config file based on the change you've made to your spec. By default, it is `false`. You should not set this to `true` if you are passing a `config_path`.

**Review by meorphis — CHANGES_REQUESTED** [review]
left one more comment inline

**Comment by cjquines** [comment]
responded to comment; also figured out why [this commit](https://github.com/stainless-sdks/cjquines-config/commit/6bcddfd4cf9fc463d7893019bd62f95f6467af97) was weird; the fix was stainless-api/stainless#6430. here's a new [app run](https://github.com/stainless-sdks/cjquines-source/actions/runs/10408781887) and [commit](https://github.com/stainless-sdks/cjquines-prod-0-config/commit/8ad6b6780d53e8c0e0fff4d7dbf18673236fd6fd), which should look better now

**Review by meorphis — APPROVED** [review]

**Pull request merged**